components/FriendPages.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import data from "@/data.json";
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledFriendBox = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 3rem;
+  flex-wrap: wrap;
+  text-align: center;
+`;
+
+export default function FriendPages() {
+  return (
+    <StyledFriendBox>
+      {data.map((friend) => (
+        <div key={friend.id}>
+          <Link href={`contacts/${friend.id}-${friend.nickname}`}>
```

Comment:
Personally, i would put the link on either the nickname or the full name. Probably the full name. If both are links, the user gets the impression that each link leads to a different page.
But that's just a preference of mine and probably nit-picking :)

Reply:
Hey thanks for the comment. Because the nickname is preceded by an individual id, contacts with the same nickname can go to different pages and the url looks funnier. I would like to leave that for now :)